README.md

```diff
@@ -44,17 +44,17 @@ This is the official Komodo sourcecode repository based on https://github.com/jl
 - Pegged Assets - Chains that maintain a peg to fiat currencies  
 - Peerchains - Scalability solution where sibling chains form a network of blockchains  
 - More in depth covered [here](https://docs.google.com/document/d/1AbhWrtagu4vYdkl-vsWz-HSNyNvK-W-ZasHCqe7CZy0)  
-- Also note you receive 5% Active User Reward on your balance.  
-[See this article for more details](https://support.komodoplatform.com/en/support/solutions/articles/29000024515-how-to-claim-the-kmd-active-user-reward-in-agama)
+- Also note you receive **0.01%** (since [KIP-0001](https://github.com/KomodoPlatform/kips/blob/main/kips/kip-0001.mediawiki) active) Active User Reward on your balance.
+[See this post for more details](https://forum.komodoplatform.com/t/komodo-active-user-reward-all-you-need-to-know/435)
```

Comment:
replace link with https://komodoplatform.com/en/docs/komodo/active-user-reward/

Reply:
Resolved in https://github.com/KomodoPlatform/komodo/pull/647/commits/db210bfffdc63e46e418fac75984b1931e06eee2 .